packages/vite/src/node/ssr/runtime/moduleCache.ts

```diff
@@ -65,6 +65,57 @@ export class ModuleCacheMap extends Map<string, ModuleCache> {
     return this.deleteByModuleId(this.normalize(fsPath))
   }
 
+  invalidate(id: string): void {
+    const module = this.get(id)
+    module.evaluated = false
+    delete module.meta
+    delete module.map
+    delete module.promise
+    delete module.exports
```

Comment:
Do we need to `delete` the props, or is it possible to only assign `= undefined`? We found in the past that `delete` has a larger perf impact. Not blocking this PR though, just wondering.

Reply:
Yeah, we can just set it to undefined here if performance is the concern